frontend/test/metabase/scenarios/smoketest/admin_setup.cy.spec.js

```diff
@@ -799,7 +799,7 @@ describe("smoketest > admin_setup", () => {
 
       // Normal user cannot make an SQL query
 
-      openNotebookEditor({ fromCurrentPage: true });
+      startNewQuestion();
```

Comment:
I've just realized that this never made sense.
Why visit a new question if we're going to sign out after that?

Anyway, not a scope for this PR. I'll deal with smoketests cleanup separately.